docs/source/main_classes/processors.mdx

```diff
@@ -12,10 +12,22 @@ specific language governing permissions and limitations under the License.
 
 # Processors
 
-This library includes processors for several traditional tasks. These processors can be used to process a dataset into
-examples that can be fed to a model.
+Processors can mean two different things in the Transformers library:
```

Comment:
Nice!

Reply:
Thanks for reworking this part of the docs!